app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
```

Comment:
- [ ] on you signup page  Sign up and log in pages
it mandatory the your  main apps and should be deployed 
* The user should be able to register in the app with full name, email and password (all mandatory).
* The user can log in the app using email and password.
* If the user is not logged in, they can't access pages that require to be logged in (all the pages described below).but when i try to login to this i am  unable to login or signup please resolve this from your user authentication

Reply:
Done